hack/python-sdk/test-sdk.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Kubernetes Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ -z `which python3` ]; then
+    apt update -y
+    apt-get install python3.10 -y
+fi
+
+if [ -z `which pip` ]; then
+    apt update -y
+    apt-get install python3-pip -y
```

Comment:
Avoiding depending on the running env would be good. But we can work on it in follow-ups.

Reply:
Makes sense. In a follow up PR I will try to download and build from source, rather than use an environment specific package manager.

Reply:
Sounds good.